arches/app/src/arches/themes/default.ts

```diff
@@ -0,0 +1,81 @@
+import { definePreset, palette } from "@primevue/themes";
+import Aura from "@primevue/themes/aura";
+
+const archesColors = Object.freeze({
+    blue: "#579ddb",
+    green: "#3acaa1",
+    red: "#f75d3f",
+});
+
+export const ArchesPreset = definePreset(Aura, {
+    primitive: {
+        arches: {
+            ...archesColors,
+            legacy: {
+                sidebar: "#2d3c4b",
+            },
+        },
+        green: palette(archesColors.green),
+        red: palette(archesColors.red),
+    },
+    semantic: {
+        primary: palette(archesColors.blue),
+        navigation: {
```

Comment:
IMO this needs to either be removed or have more descriptive naming. `arches-navigation-list` could refer to many things. `arches-navigation-list-padding` is better. same with color IMO but I feel less strongly there

Reply:
This is another one that comes from primevue theming. (Sorry, tried to clarify this by putting "custom" tokens beneath a comment, but as of now we won't have any custom tokens.) This is what we get if we do nothing:

```js
navigation
  item
    {padding: '0.5rem 0.75rem', borderRadius: '{border.radius.sm}', gap: '0.5rem'}
  list 
    {padding: '0.25rem 0.25rem', gap: '2px'}
  submenuIcon
    {size: '0.875rem'}
  submenuLabel
    {padding: '0.5rem 0.75rem', fontWeight: '600'}
```

I think leaving this in is _less_ opinionated by taking some of these paddings down to 0 or at least equal X/Y

***
On second look, the navigation-color one is custom, so I'll find a way to rename/clarify.